packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts

```diff
@@ -1202,14 +1202,11 @@ export class PaginatedMessage {
 	public addPageAction(action: PaginatedMessageAction, index: number): this {
 		if (index < 0 || index > this.pages.length - 1) throw new Error('Provided index is out of bounds');
 
-		if (!this.pageActions[index]) {
-			this.pageActions[index] = new Map<string, PaginatedMessageAction>();
-		}
-
+		this.pageActions[index] ??= new Map<string, PaginatedMessageAction>();
 		if (actionIsLinkButton(action)) {
-			this.pageActions[index]!.set(action.url, action);
+			this.pageActions[index]?.set(action.url, action);
```

Comment:
This is impossible to be nullable after the `??=` above

Reply:
If anything I'd define the collection in a variable and then use it here rather than doing the accesses again.